pkg/certificate/conditions.go

```diff
@@ -0,0 +1,111 @@
+package certificate
+
+import (
+	"time"
+
+	"github.com/openservicemesh/osm/pkg/apis/config/v1alpha2"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	ready              v1alpha2.MeshRootCertificateConditionType = "Ready"
+	accepted           v1alpha2.MeshRootCertificateConditionType = "Accepted"
+	issuingRollout     v1alpha2.MeshRootCertificateConditionType = "IssuingRollout"
+	validatingRollout  v1alpha2.MeshRootCertificateConditionType = "ValidatingRollout"
+	issuingRollback    v1alpha2.MeshRootCertificateConditionType = "IssuingRollback"
+	validatingRollback v1alpha2.MeshRootCertificateConditionType = "ValidatingRollback"
+)
+
+const (
+	trueStatus    v1alpha2.MeshRootCertificateConditionStatus = "True"
+	falseStatus   v1alpha2.MeshRootCertificateConditionStatus = "False"
+	unknownStatus v1alpha2.MeshRootCertificateConditionStatus = "Unknown"
+)
+
+const (
```

Comment:
I feel like these Reasons should probably live in the same package as the MRC resource itself

Reply:
pkg/apis/config/v1alpha2 contains primarily generated files. It is acceptable to add non-generated resources to this package?

Reply:
Move condition types to v1alpha2.

Reply:
Addressed in other comments